packages/workflow/src/workflow.ts

```diff
@@ -1133,3 +1134,52 @@ export function setHandler<Ret, Args extends any[], T extends SignalDefinition<A
     throw new TypeError(`Invalid definition type: ${(def as any).type}`);
   }
 }
+
+/**
+ * Updates this Workflow's Search Attributes by merging the provided `searchAttributes` with the existing Search
+ * Attributes, `workflowInfo().searchAttributes`.
+ *
+ * For example, this Workflow code:
+ *
+ * ```ts
+ * upsertSearchAttributes({
+ *   CustomIntField: [1],
+ *   CustomBoolField: [true]
+ * });
+ * upsertSearchAttributes({
+ *   CustomIntField: [2, 3],
+ *   CustomKeywordField: ['durable code']
+ * });
+ * ```
+ *
+ * would result in the Workflow having these Search Attributes:
+ *
+ * ```ts
+ * {
+ *   CustomIntField: [2, 3],
```

Comment:
Shouldn't this be `[1,2,3]` ? Or would the user need to take, append, then replace? If so, worth showing that example too.

Reply:
No, I'm trying to make it clear here that upsert overwrites. I'll change the numbers. In the sample, I show upserting based on current value